pkg/cli/copy.go

```diff
@@ -48,7 +50,21 @@ type ImageCopy struct {
 	client  ClientFactory
 }
 
-func (a *ImageCopy) Run(cmd *cobra.Command, args []string) error {
+func (a *ImageCopy) Run(cmd *cobra.Command, args []string) (err error) {
+	// Print a helpful error message for the user if they end up getting an authentication error
+	defer func() {
+		if err != nil {
+			var terr *transport.Error
+			if ok := errors.As(err, &terr); ok {
+				if terr.StatusCode == http.StatusForbidden {
+					logrus.Warnf("Registry authentication failed. Try running 'acorn login -l <registry>'")
+				} else if terr.StatusCode == http.StatusUnauthorized {
+					logrus.Warnf("Registry authorization failed. Ensure that you have the correct permissions to push to this registry. Run 'acorn login -l <registry>' if you have not logged in yet.")
```

Comment:
Two questions about using `logrus.Warnf` here:
- Does it print to stderr or stdout?
- Does it print out extra logrus stuff like timestamp?

I mainly ask because, for user facing error messages, I think we want to to stderr and we probably don't want timestamps.

Reply:
Good call. I don't know if it was printing to stderr or stdout, and you're right that the log-like formatting is probably not what we want to show to the user.

I replaced both of these `logrus.Warnf` calls with `fmt.Fprintln`, which allows me to plainly write to stderr.